models/Circle.js

```diff
@@ -16,6 +16,14 @@ const Circle = sequelize.define('circle', {
         validate: {
             notEmpty: { msg: 'Description should be set.' },
         },
+    },
+    gsuite_id: {
```

Comment:
Why link this to circles and not bodies as well? Might be nice to link the body accounts to the body. And use this solely for linking Google groups to circles